lib/iam_ssoe_oauth/configuration.rb

```diff
@@ -35,6 +35,7 @@ def connection
         base_path, headers: base_request_headers, request: request_options, ssl: ssl_options
       ) do |conn|
         conn.use :breakers
+        conn.use Faraday::Response::RaiseError
```

Comment:
Does this change impact the introspect call?

Reply:
It does, although an inactive response from the introspect call is a 200 and doesn't throw an error (we catch it and return unauthorized). I could add the same 500-> 502 catch there.